src/main/java/appeng/integration/IntegrationNode.java

```diff
@@ -32,16 +32,16 @@
 public final class IntegrationNode
 {
 
-	final String displayName;
-	final String modID;
-	final IntegrationType shortName;
-	IntegrationStage state = IntegrationStage.PRE_INIT;
-	IntegrationStage failedStage = IntegrationStage.PRE_INIT;
-	Throwable exception = null;
-	String name = null;
-	Class<?> classValue = null;
-	Object instance;
-	IIntegrationModule mod = null;
+	private final String displayName;
+	private final String modID;
+	private final IntegrationType shortName;
+	private IntegrationStage state = IntegrationStage.PRE_INIT;
+	private IntegrationStage failedStage = IntegrationStage.PRE_INIT;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "failedStage" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)